archon-ui-main/src/utils/copyHelpers.ts

```
@@ -0,0 +1,116 @@
+  type: CopyButtonType,
+  projectId: string,
+  itemId?: string,
+  currentView?: 'table' | 'board'
+): string => {
+  const origin = window.location.origin;
+  
+  switch (type) {
+    case 'project':
+      return `${origin}/projects/${projectId}`;
+    case 'task':
+      const view = currentView ? `?view=${currentView}` : '';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)